rfcs/text/0000-authentication-fields.md

```diff
@@ -0,0 +1,136 @@
+        "event.type": [
+            "change"
+        ],
+        "fileset.name": "activitylogs",
+        "geo.city_name": "Milton",
+        "geo.continent_name": "North America",
+        "geo.country_iso_code": "US",
+        "geo.country_name": "United States",
+        "geo.location.lat": 47.2513,
+        "geo.location.lon": -122.3149,
+        "geo.region_iso_code": "US-WA",
+        "geo.region_name": "Washington",
+        "input.type": "log",
+        "log.level": "Information",
+        "log.offset": 0,
+        "related.ip": [
+            "216.160.83.61"
+        ],
+        "service.type": "azure",
+        "source.as.number": 209,
+        "source.geo.city_name": "Milton",
+        "source.geo.continent_name": "North America",
+        "source.geo.country_iso_code": "US",
+        "source.geo.country_name": "United States",
+        "source.geo.location.lat": 47.2513,
+        "source.geo.location.lon": -122.3149,
+        "source.geo.region_iso_code": "US-WA",
+        "source.geo.region_name": "Washington",
+        "source.ip": "216.160.83.61",
+        "tags": [
+            "forwarded"
+        ]
+    }
+]
+
+Palo Alto Authentication Logs:
+FUTURE_USE, Receive Time, Serial Number, Type, Threat/Content Type, FUTURE_USE, Generated Time, Virtual System, Source IP, User, Normalize User, Object, Authentication Policy, Repeat Count, Authentication ID, Vendor, Log Action, Server Profile, Description, Client Type, Event Type, Factor Number, Sequence Number, Action Flags, Device Group Hierarchy 1, Device Group Hierarchy 2, Device Group Hierarchy 3, Device Group Hierarchy 4, Virtual System Name, Device Name, Virtual System ID, Authentication Protocol, UUID for rule, High Resolution Timestamp, Source Device Category, Source Device Profile, Source Device Model, Source Device Vendor, Source Device OS Family, Source Device OS Version, Source Hostname, Source Mac Address, Region, FUTURE_USE, User Agent, Session ID
+
+```
+## Scope of impact
+
+No impact expected as login fieldsets are not impacting any existing fields, as these proposed fields are new. Moreover, these fields allow logs to be in a greater alignment with the ECS base fields and allow for expanded utilization and wider adoption.
+
+## Concerns
+
+The concerns that might arise relate to how the nested fields could be broken out into separate fields or that fields that are arrays are numbered. There are additional potential fields that could be implemented, but until they are seen in other vendor information, it makes sense to wait to add them to ECS.
+
+## People
+
+The following are the people that consulted on the contents of this RFC.
+•	@mr1716 | author
```

Comment:
```suggestion

•	@mr1716 | author
```

nit: rendering on the same line as the previous sentence. I think markdown just needs an extra newline here to fix it.

Reply:
@ebeahan thank you very much for this input. Much appreciated

Reply:
Hey @ebeahan, what’s the next step for this? How can progress be made?